test/mocks/json-rpc-result.ts

```diff
@@ -0,0 +1,281 @@
+const MOCK_BLOCK_NUMBER = '0x1';
```

Comment:
Hey, just curious, will these response require change if data returned by CDN changes.

Reply:
not exactly sure what data you're referring to. Do you have an example? I have observed changing this value may affect tests in the `snaps/ppom-blockaid-alert`